test/integration/01-setup/01-funds-bitcoind-lnds.spec.ts

```diff
@@ -53,6 +48,7 @@ describe("Bitcoind", () => {
   })
 
   it("funds lnd1 node", async () => {
+    await waitUntilSyncAll()
```

Comment:
Can't we have just `waitUntilSync({lnds: [lnd1]})` here? I guess we can ignore whether the other lnds are synced or not because that's not relevant here. What do you think?

Reply:
Also I think this should be moved further down to #L63. I don't understand why we want it to be at the beginning of this test. Am I missing something?

Reply:
just remove it, and trust in waitUntilBlockHeight and waitUntilSync({ lnds }) of channel setup